mrp_partial_production/__init__.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+###########################################################################
+#    Module Writen to OpenERP, Open Source Management Solution
+#
+#    Copyright (c) 2013 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info@vauxoo.com
+############################################################################
+#    Coded by: Jose Morales (jose@vauxoo.com)
+#    Planified by: Nhomar Hernandez <nhomar@vauxoo.com>
+#    Audited by: Jose Morales <jose@vauxoo.com>
+############################################################################
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+from . import models
+from . import wizard
+# from . import tests
```

Comment:
Yes, the test files into the folder tests are executed automatically[1]. It is unnecessary  import the folder.

https://drive.google.com/a/vauxoo.com/file/d/0B2kzKLGF6ZvLalF0Q21CNjJ5bGc/view?usp=drivesdk